docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -84,10 +84,26 @@ This property is required; there is no default. A connection url or seeds must b
 
 As MongoDB is a document database, there is no fixed schema information in the system. So a special collection in each MongoDB database should define the schema of all tables. Please refer the :ref:`table-definition-label` section for the details.
 
-At startup, this connector tries guessing fields' types, but it might not be correct for your collection. In that case, you need to modify it manually. ``CREATE TABLE`` and ``CREATE TABLE AS SELECT`` will create an entry for you.
+At startup, this connector tries guessing fields' types as the below mapping, but it might not be correct for your collection. In that case, you need to modify it manually. ``CREATE TABLE`` and ``CREATE TABLE AS SELECT`` will create an entry for you.
```

Comment:
Maybe wrap this whole paragraph at 80 char while you are changing.. 

and use 

At startup, the connector tries to guess the data type of fields based on the mapping in the following table. 

Table

and then after the table add

The initial guess can be incorrect for your specific collection. In that case, you need to modify it manually with a ``CREATE TABLE`` or a  ``CREATE TABLE AS SELECT`` statement.

Reply:
and it might also be good to have a relevant example for such a query..

Reply:
@mosabua The CREATE TABLE and CREATE TABLE AS SELECT isn't to "modify it manually".

This should work IMO

```suggestion
At startup, the connector tries to guess the data type of fields based on the mapping in the following table. The initial guess can be incorrect for your specific collection. In that case, you need to modify it manually. Please refer the :ref:`table-definition-label` section for the details.

Creating new tables using ``CREATE TABLE`` and ``CREATE TABLE AS SELECT`` automatically create an entry for you.
```